CHANGELOG.md

```diff
@@ -21,6 +22,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Fixed
 - Fix KeyError when scroll return no hits ([#616](https://github.com/opensearch-project/opensearch-py/pull/616))
 - Fix reuse of `OpenSearch` using `Urllib3HttpConnection` and `AsyncOpenSearch` after calling `close` ([#639](https://github.com/opensearch-project/opensearch-py/pull/639))
+### Automated API Update
+- Your contribution here.
```

Comment:
> Is this line `Your contribution here` needed?

Yes, to update changelog, find and replace is used. We can change the text if required.

Reply:
You could reuse another line like "### Automated API Update". Maybe call it "Updated APIs" to match the other sections language?

Reply:
- Trying to find a way to update only in ## [Unreleased] section. 

- If "### Updated APIs" is used for finding, it will be present in changelog at several places in future after next release. 

- Action `jacobtomlinson/gha-find-replace@v3 ` will not help if `### Updated APIs` is present in several places.

```
- name: Update CHANGELOG
        uses: jacobtomlinson/gha-find-replace@v3
        with:
          find: "### Updated APIs"
          replace: "### Updated APIs\n- Updated opensearch-py to reflect the latest OpenSearch API spec."
          include: "**CHANGELOG.md"
```

Reply:
Makes sense. I think that action could use a "replace first occurrence" feature, I took a look at the project and started with adding some unit tests - https://github.com/jacobtomlinson/gha-find-replace/pull/61. Will follow up with that feature if it gets merged.